query-node/mappings/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/src/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf lib && tsc --build tsconfig.json && yarn copy-types",
+    "build": "rm -rf lib && tsc --build tsconfig.json",
     "copy-types": "cp ../../types/augment/all/defs.json lib/generated/types/typedefs.json",
```

Comment:
Why is `copy-types` no longer needed here? Is `copy-types` action still needed in some scenarios, or can it be deleted?

Reply:
This is no longer needed